charts/fluent-operator/Chart.yaml

```diff
@@ -15,7 +15,7 @@ description: A Helm chart for Kubernetes
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.4.0
+version: 2.5.0
```

Comment:
Shouldn't this be version `2.4.1` here?

Reply:
yes, should be 2.4.1
We'll change this to 2.5.0 when v2.5.0 is released